### PR TITLE
It is now possible to set the ELK version in the .env file.

### PR DESCRIPTION
**Pull request by ffknob**
I found it easier to change the version of the ELK stack this way.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,7 +66,11 @@ $ chcon -R system_u:object_r:admin_home_t:s0 docker-elk/
 
 If you're using Docker for Windows, ensure the "Shared Drives" feature is enabled for the `C:` drive (Docker for Windows > Settings > Shared Drives). See [Configuring Docker for Windows Shared Drives](https://blogs.msdn.microsoft.com/stevelasker/2016/06/14/configuring-docker-for-windows-volumes/) (MSDN Blog).
 
-## Usage
+## Getting started
+
+### Version
+
+Change the ELK stack version in the _.env_ file.
```
Comment (antoineco) [inline README.md]: I'd rather move that to the customization section. The easiest would be to revert this: https://github.com/deviantony/docker-elk/commit/8f5b242d6a51615899ae920cc420b6393853ebcd

**Comment by antoineco** [comment]
Good idea! Fast forward 8 months, I believe we can now assume users should have a recent enough Docker version. @deviantony, concerns?

**Comment by deviantony** [comment]
Sounds good to me, maybe rename `TAG` to something more explicit.

**Comment by ffknob** [comment]
How about ELK_TAG instead of TAG?
Or do you think it makes sense to have differente TAG variable to the different Elastic product (ELASTICSEARCH_TAG, LOGSTASH_TAG, KIBANA_TAG, ...) so it will be possible to have different versions for different products...?

**Comment by deviantony** [comment]
I believe that we should use the same version for each component of the stack, this is the way Elastic wants their stack to be used. I was just thinking to rename it to something else like `VERSION` as it's  more explicit IMO.

Thoughts @antoineco

**Comment by antoineco** [comment]
I actually liked the original commit by the original author :)
https://github.com/deviantony/docker-elk/commit/8f5b242d6a51615899ae920cc420b6393853ebcd
Why not just reverting it?

**Comment by deviantony** [comment]
@antoineco Agree

**Comment by antoineco** [comment]
Commit reverted, merging.

**Pull request merged**